backend/src/seed/factories/users.js

```diff
@@ -24,7 +24,7 @@ export default function create() {
       }
       args = await encryptPassword(args)
       const user = await neodeInstance.create('User', args)
-      const email = await neodeInstance.create('EmailAddress', { email: args.email })
+      const email = await factoryInstance.create('EmailAddress', { email: args.email })
```

Comment:
what's the difference between `factoryInstance` and `neodeInstance`?

Reply:
We might want to refactor this with https://github.com/aexmachina/factory-girl (although the repo looks rather old).

`factoryInstance` is a reference to the other factories. If you would do `factoryInstance.create(..)` that would give you all the defaults and also setup associations. If you use `neodeInstance` you would end up with a `User` without `EmailAddress` etc.

Reply:
ahhh, ok! great... thanks